src/dmtcp_restart.cpp

```diff
@@ -345,6 +349,20 @@ static void runMtcpRestart(int is32bitElf, int fd, ProcessInfo *pInfo)
   sprintf(fdBuf, "%d", fd);
   sprintf(stderrFdBuf, "%d", PROTECTED_STDERR_FD);
 
+#ifdef HAS_PR_SET_PTRACER
+  if (getenv("DMTCP_GDB_ATTACH_ON_RESTART")) {
+    JNOTE("\n     *******************************************************\n"
```

Comment:
JASSERT_STDERR would have been a better choice here instead of JNOTE. It would have looked like:

JASSERT_STDERR << "....." << getpid();

However, since we don't have a uniform policy about it, let's commit it as is.